consensus/safety-rules/benches/safety_rules.rs

```diff
@@ -114,14 +114,51 @@ fn thread(n: u64) {
     lsr(safety_rules_manager.client(), signer, n);
 }
 
+fn vault(n: u64) {
+    let signer = ValidatorSigner::from_int(0);
+    let waypoint = test_utils::validator_signers_to_waypoint(&[&signer]);
+
+    let mut storage =
+        VaultStorage::new(VAULT_HOST.to_string(), VAULT_TOKEN.to_string(), None, None);
+    storage.reset_and_clear().unwrap();
+
+    let storage = PersistentSafetyStorage::initialize(
+        Storage::from(storage),
+        signer.author(),
+        signer.private_key().clone(),
+        Ed25519PrivateKey::generate_for_testing(),
+        waypoint,
+    );
+    let safety_rules_manager = SafetyRulesManager::new_thread(storage, false);
+    lsr(safety_rules_manager.client(), signer, n);
+}
+
 pub fn benchmark(c: &mut Criterion) {
     let count = 100;
+    let duration_secs = 5;
+    let samples = 10;
+
+    let storage = VaultStorage::new(VAULT_HOST.to_string(), VAULT_TOKEN.to_string(), None, None);
+
+    let enable_vault = if storage.available().is_err() {
+        println!("Vault is not availble");
```

Comment:
```suggestion
        println!("Vault is not available at {}", VAULT_HOST.to_string());
```

Reply:
Since you are anyway making changes, maybe one more slight change.

println!("Vault is not available at {}, so Vault will not be benchmarked", VAULT_HOST.to_string());